src/middlewares/multipartFormDataParser.js

```diff
@@ -0,0 +1,49 @@
+const BusBoy = require('busboy')
+const contentTypeLib = require('content-type')
+const createError = require('http-errors')
+
+module.exports = () => ({
+  before: async (handler, next) => {
```

Comment:
Since this middleware should be compatible with Node6 (still not deprecated in middy v0.x and won't probably even in middy 1.0.x). We should not use directly `async`/`await` in middlewares. If a user will try to use this middleware in Node 6 they would have a failure

Reply:
Good point, removed.

Reply:
Thanks :)